java-checks-test-sources/src/main/files/non-compiling/checks/EscapedUnicodeCharactersTextBlocksCheck.java

```diff
@@ -0,0 +1,116 @@
+package checks;
+
+class EscapedUnicodeCharactersCheck {
+  void method() {
+    String prefix = """
+      n\uuuu00E9e"""; // Noncompliant@-1 [[sc=21;ec=20]] {{Remove this Unicode escape sequence and use the character instead.}}
```

Comment:
We should probably include examples where multiple line breaks follow one another.
```java
    String prefix = """


      n\uuuu00E9e

""";
```